libr/io/meson.build

```diff
@@ -9,6 +9,7 @@ r_io_sources = [
   'undo.c',
   'p_cache.c',
   'p/io_ar.c',
+  'p/io_fd.c',
```

Comment:
this is not enough at the moment to enable a plugin in meson. Add it to the `io_plugins` array in `libr/meson.build`.

Reply:
Thanks for the tip! i missed that